pkg/obfuscate/obfuscate.go

```diff
@@ -148,6 +148,19 @@ type SQLConfig struct {
 	// Valid values are "obfuscate_only", "obfuscate_and_normalize"
 	ObfuscationMode ObfuscationMode `json:"obfuscation_mode" yaml:"obfuscation_mode"`
 
+	// RemoveSpaceBetweenParentheses specifies whether to remove spaces between parentheses.
+	// By default, spaces are inserted between parentheses during normalization.
+	RemoveSpaceBetweenParentheses bool `json:"remove_space_between_parentheses" yaml:"remove_space_between_parentheses"`
```

Comment:
It looks like these new options are only relevant if the user has configured the `ObfuscationMode` specifically to use the sqllexer package? Should there be more clarity in these comments + the release note to make this clear? Also there will likely need to be a docs update for the datadog agent security page

Reply:
I will update comments and release to make it clear that these options are only effective when ObfuscationMode is configured. 
Regarding doc update on `datadog agent security page`, is it because we are potentially changing the obfuscation behavior?

Reply:
cool. I updated the comments + release note to state when these options are relevant.